Pipfile

```diff
@@ -20,6 +20,7 @@ sphinx-last-updated-by-git = "*"
 sphinx-rtd-theme = "*"
 
 [packages]
+feedparser = "*"
```

Comment:
```suggestion
```
Please add dependencies which are required on the production server to [setup.py](https://github.com/Integreat/integreat-cms/blob/2c5d505fd975819e461b0ab5cc24e18b7c0473a2/setup.py#L30). At the moment, the Pipfile is just for dev dependencies.